packages/forklift-console-plugin/src/modules/Plans/views/details/tabs/VirtualMachines/Plan/PlanVirtualMachinesList.tsx

```diff
@@ -79,17 +77,11 @@ export const PlanVirtualMachinesList: FC<{ obj: PlanData }> = ({ obj }) => {
   }));
   const vmDataSource: [VMData[], boolean, unknown] = [vmData || [], true, undefined];
   const vmDataToId = (item: VMData) => item?.specVM?.id;
-  const canSelect = (item: VMData) =>
-    item?.statusVM?.started === undefined || item?.statusVM?.error !== undefined;
-  const onSelect = () => undefined;
-  const initialSelectedIds = [];
 
-  const actions: PageGlobalActions = [
-    ({ selectedIds }) => <PlanVMsDeleteButton selectedIds={selectedIds || []} plan={plan} />,
-  ];
+  const actions: PageGlobalActions = [() => <PlanVMsEditButton plan={plan} />];
```

Comment:
please consider useMemo as it will recreate each render

Reply:
In this case, using useMemo wouldn't be beneficial for optimizing performance, since the logic for creating the actions array is quite simple and doesn't involve any expensive computations.

But I can pass it directly into the component actions prop